.claude/scripts/check-codex-lane-readiness.sh

```
@@ -6,7 +6,7 @@ set -uo pipefail
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 REPO_ROOT="$(cd "$SCRIPT_DIR/../.." && pwd)"
 
-CODEX="${CODEX:-/Applications/Codex.app/Contents/Resources/codex}"
+CODEX="$($SCRIPT_DIR/resolve-codex-binary.sh 2>/dev/null || true)"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)